tests/test_config.py

```diff
@@ -0,0 +1,49 @@
+"""
+Tests for Ralph's configuration loading
+"""
+
+from os import environ
+
+import ralph.defaults
+from ralph.defaults import config
+
+
+# pylint: disable=invalid-name, unused-argument
+def test_config_default(fs, monkeypatch):
+    """Test that when no value is given, the default parameter is chosen"""
+
+    def config_mock():
+        return {}
+
+    monkeypatch.setattr(ralph.defaults, "load_config", config_mock)
+    assert environ.get("RALPH_HISTORY_FILE", None) is None
+
+    assert config("RALPH_HISTORY_FILE", "history_default") == "history_default"
+
+
+# pylint: disable=invalid-name, unused-argument
+def test_config_file(fs, monkeypatch):
+    """Test that when the environment and command-line arguments don't specify a
+    value, the value from the config file is taken instead"""
+
+    def config_mock():
+        return {"RALPH_HISTORY_FILE": "history_config_file"}
+
+    monkeypatch.setattr(ralph.defaults, "load_config", config_mock)
+    assert environ.get("RALPH_HISTORY_FILE", None) is None
+
+    assert config("RALPH_HISTORY_FILE", "history_default") == "history_config_file"
+
+
+# pylint: disable=invalid-name, unused-argument
+def test_config_env(fs, monkeypatch):
+    """Test that when the environment specifies a value, it is chosen instead of
+    the configuration file's value"""
+
+    def config_mock():
+        return {"RALPH_HISTORY_FILE": "history_config_file"}
+
+    monkeypatch.setattr(ralph.defaults, "load_config", config_mock)
+    monkeypatch.setenv("RALPH_HISTORY_FILE", "history_env")
+
+    assert config("RALPH_HISTORY_FILE", "history_default") == "history_env"
```

Comment:
This file should be named: `test_defaults.py`, it's a convention since you are testing the `defaults.py` module.

Reply:
I didn't know about this convention, thanks
Fixed